docs/tracing.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Tracing"
+date: 2020-05-13T12:09:00+01:00
+weight: 55
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs
+geekdocFilePath: tracing.md
+---
+
+We use [Jaeger](https://www.jaegertracing.io) for request tracing within oCIS. There are only a few steps required
+to get started:
+
+1. Start oCIS with tracing enabled: `REVA_TRACING_ENABLED=true bin/ocis server`. This will tell services to send
```

Comment:
Start `ocis-reva` in this case

Reply:
:thinking: oCIS single binary starts all the reva processes and looking into jaeger ui, there are traces showing up from reva. Feels like an easier way than firing up all the ocis-reva processes individually.

Reply:
Would be cool though, to add details on `OCIS_TRACING_ENABLED` as well and highlight the differences.

Reply:
correct. What I mean is: you start the single binary, but using this flag will enable tracing only for the reva subcommands.

Reply:
just wording.

Reply:
To be honest, I thought that we only had reva tracing. Looked up the `OCIS_TRACING_ENABLED` env var after your comment. Unfortunately it doesn't have a default jaeger endpoint set. I will look up the endpoint and change the doc to using `OCIS_TRACING_ENABLED`. Why would one trace only reva services, when we have tracing for all services. ;-)

Reply:
proxy should have tracing too

Reply:
https://github.com/owncloud/ocis-proxy/blob/master/pkg/flagset/flagset.go#L61

Reply:
You could have the assumption, that `OCIS_TRACING_ENABLED` will switch on tracing for all services. But you need to enable them one by one.

Reply:
that was indeed my assumption. Thanks for clarifying!

Reply:
I changed the documentation to incorporate that every service has it's own set of flags for tracing configuration. Please check again.